daml-foundations/daml-ghc/daml-compiler/src/DA/Service/Daml/Compiler/Impl/Handle.hs

```diff
@@ -171,7 +157,7 @@ buildDar service file mbExposedModules pkgName sdkVersion buildDataFiles dalfInp
         []
         pkgName
         sdkVersion
-    else actionToExceptT service $ runMaybeT $ do
+    else runAction service $ runMaybeT $ do
```

Comment:
I'm starting to wonder if `runAction` should always convert `BadDependency` into a `Nothing` on its own? Or have `runActionMaybe` to encapsulate that behaviour?

Reply:
Sounds reasonable but I also don’t really mind the `MaybeT` stuff now that I’ve gotten used to it.

Reply:
It's irking me more every day... I guess the tables have turned. If it irks me enough I'll do the needful.